Zend/zend_execute.c

```diff
@@ -2200,7 +2200,7 @@ static zend_always_inline zval *zend_fetch_dimension_address_inner(HashTable *ht
 		}
 str_index:
 		if (type != BP_VAR_W) {
-			retval = zend_hash_find_ex(ht, offset_key, ZEND_CONST_COND(dim_type == IS_CONST, 0));
+			retval = zend_hash_find(ht, offset_key, ZEND_CONST_COND(dim_type == IS_CONST));
```

Comment:
Typo here, the `, 0` was part of the ZEND_CONST_COND, not the zend_hash_find_ex.